tldw_Server_API/app/api/v1/endpoints/media/process_mediawiki.py

```
@@ -0,0 +1,243 @@
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Dump file has no filename.",
+        )
+
+    with TempDirManager(prefix="mediawiki_ingest_", cleanup=False) as temp_dir:
+        temp_dir_path = Path(temp_dir)
+        temp_file_path = temp_dir_path / sanitize_filename(dump_file.filename)
+        try:
+            async with aiofiles.open(temp_file_path, "wb") as f:
+                content = await dump_file.read()
+                await f.write(content)
+        except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.5)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)